src/client/features/search/index.js

```diff
@@ -11,6 +11,14 @@ const Icon = require('../../common/components').Icon;
 const { ServerAPI } = require('../../services');
 const Landing = require('./landing-box');
 
+//File in the scripts folder which has all the information needed to associate each URL with an ID
+//load that file and create a mapping object
+const URLData = require('../../../scripts/datasources/datasources');
```

Comment:
I am pretty sure the datasource info is already being loaded in the component.

See: https://github.com/PathwayCommons/app-ui/pull/745/files#diff-a1f9f779cabf8780553a2f0a90378e43L40

Reply:
I think the `dataSources` in the code you linked is coming from a file like:
http://apps.pathwaycommons.org/pc-client/datasources (live)
http://localhost:3000/pc-client/datasources (local version)
I'm not sure where that page is being created, and modifying the `datasources.js` file doesn't appear to change it.  It also doesn't contain the `homePage` property from the `datasources.js` file which I need.  Any advice?